pkgs/development/libraries/qt-6/modules/qtquick3d.nix

```diff
@@ -0,0 +1,12 @@
+{ qtModule
+, qtbase
+, qtdeclarative
+, openssl
+}:
+
+qtModule {
+  pname = "qtquick3d";
+  qtInputs = [ qtbase qtdeclarative ];
+  buildInputs = [ openssl openssl.dev ];
```

Comment:
openssl.out or openssl.bin? You are sure this works for cross compile? This looks weird.